### DISCUSSION TITLE
Dịch từ "reproduce"/"reproducibility"

### DISCUSSION THREAD

**Discussion by quangvnai** [d2l-ai/d2l-vi#2690] [Terminology]
Từ này khá phổ biến và đang dần trở thành một trong những điều kiện khi nộp báo.
Ở [1 PR](https://github.com/aivivn/d2l-vn/pull/2612/files), thì bạn Phát dịch là **tái hiện**. **Tái hiện** hay **tái xuất** có vẻ trùng với nghĩa của từ "reappear" hơn, (nghe có phần giống showbiz nhỉ :))).

Bản thân từ này đã được [dịch trong Toán](http://tratu.soha.vn/dict/en_vn/reproduce) là **tái lập**.
* reproduce: tái lập
* reproducibility: khả năng tái lập (kết quả).

Mọi người nghĩ sao?

cc: @minhduc0711 @cuongvng @lkhphuc @rootonchair

**Comment by cuongvng** [comment]
em thấy `tái lập` ok.

**Comment by lkhphuc** [comment]
từ `restore/re-establish` cũng hay được dịch là `tái lập` rồi.
`tái sản xuất` là sát và độc nhất nhưng mình thấy không hay, và cũng không liên quan nhiều tới "các thí nghiệm".

Mình thấy `tái tạo` hợp hơn xíu, đang hay được dịch từ `recreate, reproduce, regenerate`.

**Reply by quangvnai** [reply]
yeah, từ tái tạo nghe chuẩn chỉnh hơn đấy.
